lib/FCM/Util/ConfigReader.pm

```diff
@@ -310,7 +310,6 @@ sub _parse_fcm2_var {
                 $E->CONFIG_VAR_UNDEF, $state_ref->{ctx}, "undef($symbol)",
             );
         }
-        $substitute ||= q{};
```

Comment:
I'd never seen this construct before, so I've learnt something from sanity-checking this!